docker/base/Dockerfile

```diff
@@ -21,7 +21,7 @@
 # TODO(mberlin): Remove the symlink and this note once
 # https://github.com/docker/hub-feedback/issues/292 is fixed.
 
-ARG bootstrap_version=13
+ARG bootstrap_version=13.2
```

Comment:
Why are we using a `.2` version on main? We should be using 14.

Reply:
This is solved by e816bdcda3